DESCRIPTION

```diff
@@ -38,6 +38,7 @@ LazyData: true
 Roxygen: list(markdown = TRUE)
 RoxygenNote: 7.2.3
 Depends: R (>= 4.0)
+Remotes: pharmaverse/admiraldev
```

Comment:
@ddsjoberg is this all I would have to do? or does I need to remove the imports line too?

Reply:
You'll want to make one small change to include the min version number required.

```
Remotes: pharmaverse/admiraldev (>= 0.5.0.9000)
```

Then if anyone does any dev work, like redocumenting, and they don't have this version installed, they'll get a note that they need to install the updated version.